.changes/v1.0.0/29-notes.md

```diff
@@ -0,0 +1 @@
+- Use `MetaContainer` to store provider SDK Clients, including `govcd.VCDClient` [GH-29]
```

Comment:
`Container` is too abstract, IMHO. What about naming it `AbstractClient`, `GenericClient` or `ClientContainer`?

Reply:
But the `MetaContainer` is not a client exactly - it is just a "place" to put clients. At the moment we only have one, but the whole point is to have a "place to put more":
https://github.com/vmware/terraform-provider-vcfa/pull/29/files#diff-a120c9841d0e5131e9973a5cefc4c68aa551781b32e07d93c53adb2466656e80R224-R226

Reply:
Based on this I could say perhaps the `ClientContainer` could be the only that fits it

Reply:
I like `ClientContainer`. It seems `meta` is like a keyword as seen in https://github.com/hashicorp/terraform-provider-scaffolding/blob/main/internal/provider/resource_scaffolding.go, but maybe `ClientContainer` is more meaningful for us?

Reply:
Redoing to `ClientContainer` as it looks to be common ground

Reply:
Changed.